docs/diagrams/FindClassDiagram.puml

```diff
@@ -0,0 +1,22 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+Class "FindCommand" as FindCommand
+Command <|-- FindCommand
+FindCommand --righto "1" Predicate : contains
+FindPersonDescriptor .up. FindCommand
+
+Name "*" *-up- FindPersonDescriptor
+Phone "*" *-up- FindPersonDescriptor
+Address "*" *-up- FindPersonDescriptor
+Tag "*" *-up- FindPersonDescriptor
```

Comment:
Should the composition be on the other side instead? The descriptor is the "whole" in this whole-part relationship.